Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -716,16 +716,15 @@
     (sysctl-name-prefix "hw.perflevel") ;; <rdar://problem/76782530>
 )
 
-(deny sysctl-read (with no-report)
-    (sysctl-name "vm.task_no_footprint_for_debug"))
-
 #if HAVE(SANDBOX_STATE_FLAGS)
 (with-filter (require-not (state-flag "WebContentProcessLaunched"))
     (allow sysctl-read (sysctl-name "vm.malloc_ranges"))) ;; <rdar://problem/105161083>
 #else
 (allow sysctl-read (sysctl-name "vm.malloc_ranges")) ;; <rdar://problem/105161083>
 #endif
 
+(deny sysctl-write (with no-report) (with telemetry))
```

Comment:
This doesn't seem relevant to this particular bug report. Was this new telemetry you wanted to capture for another bug?

Reply:
This was an intentional change, since I think it would be good if we could inspect telemetry for which sysctls are being denied without having local reports.

Thanks for reviewing!